Formula/e/ente-cli.rb

```diff
@@ -0,0 +1,22 @@
+class EnteCli < Formula
+  desc "Utility for exporting data from Ente and decrypt the export from Ente Auth"
+  homepage "https://github.com/ente-io/ente/tree/main/cli"
+  url "https://github.com/ente-io/ente/archive/refs/tags/cli-v0.2.2.tar.gz"
+  sha256 "d03754880fe7dfdc422b37e4864c2aa469a2b19c645346de2d2de7fa62b71de3"
+  license "AGPL-3.0-only"
+  head "https://github.com/ente-io/ente.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cli" do
+      system "go", "build", *std_go_args(ldflags: "-s -w"), "main.go"
+      bin.install_symlink "ente-cli" => "ente"
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/ente version 2>&1")
+    assert_match "Version #{version}", output
```

Comment:
```suggest
    assert_match version.to_s, shell_output("#{bin}/ente version")
```

And you need to add a test for real usage. BTW, it pops up that cannot find a keychain for ente.

Reply:
> cannot find a keychain for ente
Yeah, it's only during the test (in the contained brew env) and it's already fixed on head (see the [PR](https://github.com/ente-io/ente/pull/4028)), so should be released soon.

> version.to_s
I'm not sure to understand. The output is `Version x.xx`., so the `version.to_s` does not seem to fit.

Reply:
`assert_match` is not `assert_equal` and it will check whether the first string is in the second one.

Reply:
ok thanks. (First lines of Ruby here, I've basically taken inspiration from other formulae)

Reply:
> And you need to add a test for real usage.

The issue is that for other commands than `help` and `version`, the cli is looking for a keychain, and it can't access it during the brew tests.